src/plugins/dashboard/public/application/dashboard_strings.ts

```diff
@@ -56,3 +56,129 @@ export function getDashboardTitle(
 
   return displayTitle;
 }
+
+export const dashboardBreadcrumb = i18n.translate('dashboard.dashboardAppBreadcrumbsTitle', {
+  defaultMessage: 'Dashboard',
+});
+
+export const dashboardPageTitle = i18n.translate('dashboard.dashboardPageTitle', {
+  defaultMessage: 'Dashboards',
+});
+
+export const dashboard60Warning = i18n.translate('dashboard.urlWasRemovedInSixZeroWarningMessage', {
+  defaultMessage: 'The url "dashboard/create" was removed in 6.0. Please update your bookmarks.',
+});
+
+export const dashboardListingTable = {
+  entityName: i18n.translate('dashboard.listing.table.entityName', {
+    defaultMessage: 'dashboard',
+  }),
+  entityNamePlural: i18n.translate('dashboard.listing.table.entityNamePlural', {
+    defaultMessage: 'dashboards',
+  }),
+  tableListTitle: i18n.translate('dashboard.listing.dashboardsTitle', {
+    defaultMessage: 'Dashboards',
+  }),
+  tableCaption: i18n.translate('dashboard.listing.dashboardsTitle', {
+    defaultMessage: 'Dashboards',
+  }),
```

Comment:
you're repeating the same keys and values... this is where the dictionary of methods comes in: you can audit the usage of a single key without having to search for the key.

```
const getDashboardsTitle = () => 
  i18n.translate('dashboard.listing.dashboardsTitle', {
    defaultMessage: 'Dashboards',
  });

// ...

getTableListTitle: () => getDashboardsTitle(),
getTableCaption: () => getDashboardsTitle(),
```

Reply:
This is cool, I've implemented this suggestion.